src/common/router.js

```diff
@@ -12,15 +12,20 @@ function parse(pathInfo) {
 }
 
 export const route = {};
-export const lastRoute = {};
+
+Object.defineProperties(route, {
+  stack: { value: [] },
+  last: { get: () => route.stack[route.stack.length - 1] || {} },
```

Comment:
`route.last` can be calculated as needed or a computed property powered by Vue, so I don't think we need `defineProperties` here, just `const route = { stack: [] }` is enough.

Reply:
A normal property will be copied by Object.assign so I would have to declare a separate variable for `stack`. Should I do it anyway?

Reply:
I see, it makes sense then.
Normally I would explicitly pick the fields that I care so that no other data will be stored unexpectedly.

Reply:
I also tried a separate variable for `stack` (not even exported) and a separate exported function like `lastRoute`, which is also fine by me, but I just wanted to expose the single `route` entity. Anyway I don't know how to organize this better so I'm all ears.